utils/src/tree/isAzExtParentTreeItem.ts

```diff
@@ -0,0 +1,19 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.txt in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import type { AzExtParentTreeItem } from "./AzExtParentTreeItem";
+import type { AzExtTreeItem } from "./AzExtTreeItem";
+import type { IAzExtParentTreeItemInternal } from "./InternalInterfaces";
+
+/**
+ * Using instanceof AzExtParentTreeItem causes issues since each extension has their own version of the utils. Instead, check _isAzExtParentTreeItem
+ */
+export function isAzExtParentTreeItem(maybeParentTreeItem: unknown): maybeParentTreeItem is AzExtParentTreeItem {
```

Comment:
I'd actually prefer renaming the file to `isAzExtTreeItem` and flipping the order around, since `AzExtTreeItem` is the base class, and `AzExtParentTreeItem` is the inheriting class.

Reply:
Good suggestion, thanks